api/v3/MailingEventQueue.php

```diff
@@ -85,6 +85,10 @@ function civicrm_api3_mailing_event_queue_delete($params) {
  *   Array of parameters determined by getfields.
  */
 function _civicrm_api3_mailing_event_queue_create_spec(&$params) {
-  $params['job_id']['api.required'] = 1;
+  $params['job_id'] = array(
```

Comment:
Ah - this is causing a test failure because that one is a real field. Not sure why it even seemed to need fixing - perhaps it was just because all the other mailing ones were pseudo-entities I thought that was